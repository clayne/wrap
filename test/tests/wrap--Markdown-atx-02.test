wrap | /dev/null | -u | md-atx-02.md | 0
