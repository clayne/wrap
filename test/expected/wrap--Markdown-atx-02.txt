# Installation Instructions
Copying and distribution of this file, with or without modification, are
permitted in any medium without royalty provided the copyright notice and this
notice are preserved.  This file is offered as-is, without warranty of any
kind.

## Basic Installation
Briefly, the shell commands `./configure; make; make install' should configure,
build, and install this package.  The following more-detailed instructions are
generic; see the `README' file for instructions specific to this package.  Some
packages provide this `INSTALL' file but do not implement all of the features
documented below.  The lack of an optional feature in a given package is not
necessarily a bug.  More recommendations for GNU packages can be found in *note
Makefile Conventions: (standards)Makefile Conventions.
